packages/android_alarm_manager/android/gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.jvmargs=-Xmx1536M
+android.enableR8=true
```

Comment:
I don't think we want this as part of this patch

Reply:
it's good, I took care of all the comments